### PR TITLE
Replace multiply_ratio with checked_mul_floor and checked_mul_floor

### PR DESCRIPTION
**Pull request by lubkoll**

### CODE REVIEW

File: smart-contracts/osmosis/contracts/cl-vault/src/helpers/msgs.rs [hunk 1]
```diff
@@ -114,15 +100,15 @@ fn osmosis_swap_exact_amount_in_msg(
 }
 
 fn cw_dex_execute_swap_operations_msg(
-    dex_router_address: Addr,
```
Comment (LaurensKubat) [inline smart-contracts/osmosis/contracts/cl-vault/src/helpers/msgs.rs:117]: This should be reverted so we keep type safety for callers of this function

**Review by LaurensKubat — APPROVED** [review]
lgtm except on change from Addr to String

File: smart-contracts/osmosis/contracts/cl-vault/src/helpers/msgs.rs (hunk 1 above)
Comment (lubkoll) [reply smart-contracts/osmosis/contracts/cl-vault/src/helpers/msgs.rs]: done

**Review by LaurensKubat — APPROVED** [review]
lgtm

**Pull request merged**